src/compression.rs

```diff
@@ -193,7 +193,7 @@ pub const SUPPORTED_COMPRESSION_METHODS: &[CompressionMethod] = &[
 
 pub(crate) enum Decompressor<R: io::BufRead> {
     Stored(R),
-    #[cfg(feature = "_deflate-any")]
+    #[cfg(feature = "deflate-flate2")]
```

Comment:
Change this to `feature = "flate2"` throughout, so that `deflate-zlib` will also enable it.

Reply:
It should be fine. `deflate-zlib` implies the `deflate-flate2` feature. Tested with only `features = ["deflate-zlib"]` set as well.